src/test/scala/no/ndla/imageapi/integration/MappingApiClientTest.scala

```diff
@@ -1,97 +0,0 @@
-/*
```

Comment:
Er det andre tester som tar med seg lisens og språk? Hvis ikke fortjener det vel en test eller to...